win32ss/drivers/miniport/xboxvmp/xboxi2c.c

```diff
@@ -0,0 +1,123 @@
+/*
+ * PROJECT:     ReactOS Xbox miniport video driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     I2C SMBus routines
+ * COPYRIGHT:   Copyright 2004 Gé van Geldorp
+ *              Copyright 2004 Filip Navara
+ *              Copyright 2019 Stanislav Motylkov (x86corez@gmail.com)
+ */
+
+/* INCLUDES *******************************************************************/
+
+#include "xboxvmp.h"
+
+#include <debug.h>
+#include <dpfilter.h>
+
+/* PUBLIC AND PRIVATE FUNCTIONS ***********************************************/
+
+static
```

Comment:
For those of us who like to verify that copied code is unchanged:
`git diff -w -C15 xboxvmp^..xboxvmp -- win32ss/drivers/miniport/xboxvmp/xboxvmp.c win32ss/drivers/miniport/xboxvmp/xboxi2c.c`
(the answer is, it is)

Reply:
Why is there copied code?

Reply:
See discussion above: https://github.com/reactos/reactos/pull/1896#discussion_r320049128 :slightly_smiling_face: